CONTRIBUTING.md

```diff
@@ -38,20 +38,20 @@ you should read it.
 
 ### The textbook
 
-:mag: [Developer wiki](https://github.com/rangle/batarangle/wiki)
+:mag: [Developer wiki](https://github.com/rangle/augury/wiki)
 
 If you want to get technical, we've got you covered too. We have [a
-README just for developers](https://github.com/rangle/batarangle/wiki). Jumping into a new codebase
+README just for developers](https://github.com/rangle/augury/wiki). Jumping into a new codebase
 can be daunting so we've tried, as best as we could, to give you an
-overview into the design of Batarangle and our intentions. It's a
+overview into the design of Augury and our intentions. It's a
 glimpse into our heads, if you will. We're not yet sure that's a good
 thing but we can assure you it's a useful one.
 
 ### The chat room
 
 :speech_balloon: [Slack](http://batarangle-slack.herokuapp.com/)
```

Comment:
yes @bertrandk needs to change heroku app as well as domain etc. working on those still